src/arena.c

```diff
@@ -1670,6 +1686,11 @@ arena_new(tsdn_t *tsdn, unsigned ind, const arena_config_t *config) {
 	arena_set(ind, arena);
 	arena->ind = ind;
 
+	/* Init the name. */
+	sprintf(arena->name, "%s_%u", arena_is_auto(arena) ? "auto" : "manual",
+	    arena->ind);
+	arena->name[ARENA_NAME_LEN - 1] = '\0';
```

Comment:
Although the arena's index cannot be too large and we should be completely safe, added this line just to make it safe.